types/agent.go

```diff
@@ -81,6 +81,10 @@ type AgentsParams struct {
 	Tags   *string
 }
 
+func (p *AgentsParams) TagFilter(s string) {
```

Comment:
I don't know about these helpers 🤔 
We are not doing it like thise anywhere else.

I'm not against it, but also we should be consistent.

Reply:
I don't think we need them, as we have parameters already for this. Maybe in the future when we compose more complex tag queries

Reply:
i got from the issue example, should i remove so? 🤔